<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Epi-Gimp</string>
  </property>
    <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
         <layout class="QHBoxLayout" name="horizontalCenterLayout">
            <item>
             <spacer name="horizontalSpacerLeft">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Expanding</enum>
                </property>
             </spacer>
            </item>
            <item alignment="Qt::AlignCenter">
             <widget class="Canvas" name="canvas">
                <property name="minimumSize">
                 <size>
                    <width>800</width>
                    <height>600</height>
                 </size>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacerRight">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Expanding</enum>
                </property>
             </spacer>
            </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacerBottom">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
         </spacer>
        </item>
     </layout>
    </widget>
  <widget class="QToolBar" name="toolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <widget class="QWidget" name="toolBarWidget">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="colorLabel">
       <property name="text">
        <string>Couleur:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="colorButton">
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
      </spacer>
     </item>
  <!-- brush size controls removed (replaced by popup) -->
    </layout>
   </widget>
  </widget>
   <widget class="QToolBar" name="leftToolBar">
    <attribute name="toolBarArea">
      <enum>LeftToolBarArea</enum>
    </attribute>
     <widget class="QWidget" name="leftToolBarWidget">
        <layout class="QVBoxLayout" name="verticalToolLayout">
      <item>
       <layout class="QHBoxLayout" name="row1Layout">
        <item>
         <widget class="QPushButton" name="toolBrushButton">
          <property name="toolTip">
           <string>Brush</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="toolColorButtonLeft">
          <property name="toolTip">
           <string>Color</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="row2Layout">
        <item>
         <widget class="QPushButton" name="toolBucketButton">
          <property name="toolTip">
           <string>Bucket Fill</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="toolSizeButton">
          <property name="toolTip">
           <string>Brush Size</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="row3Layout">
        <item>
         <widget class="QPushButton" name="toolEraserButton">
          <property name="toolTip">
           <string>Eraser</string>
          </property>
         </widget>
        </item>
            <item>
             <widget class="QPushButton" name="toolImageButton">
               <property name="toolTip">
                <string>Ins√©rer une image</string>
               </property>
             </widget>
            </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="row4Layout">
        <item>
         <widget class="QPushButton" name="toolMoveButton">
          <property name="toolTip">
           <string>Move</string>
          </property>
         </widget>
        </item>
            <item>
             <widget class="QPushButton" name="toolAreaButton">
               <property name="toolTip">
                <string>Area Select</string>
               </property>
             </widget>
            </item>
       </layout>
      </item>
         <item>
           <spacer name="toolsVerticalSpacer">
            <property name="orientation">
              <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
            </property>
           </spacer>
         </item>
        </layout>
      </widget>
   </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
